src/app/authentication/constants/routes.go

```diff
@@ -1 +1,7 @@
-package constants
\ No newline at end of file
+package constants
+
+
+const (
+	CustomerSignupEndpoint = "/customer-signup"
+	CustomerSigninEndpoint = "/customer-signin"
+)
```

Comment:
```suggestion
package constants

const (
	CustomerSignupEndpoint = "/customer-signup"
	CustomerSigninEndpoint = "/customer-signin"
)
```
Single space is more than enough. Additional spaces increase the lines of code.